glib/src/unichar.rs

```diff
@@ -0,0 +1,238 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use std::mem::MaybeUninit;
+
+use crate::{
+    translate::{from_glib, IntoGlib},
+    UnicodeBreakType, UnicodeScript, UnicodeType,
+};
+
+mod sealed {
+    pub trait Sealed: Into<u32> {
+        fn from_unichar(unichar: u32) -> char;
```

Comment:
This doesn't make much sense. The `Sealed` trait is only for trait privacy reasons, it should not contain any functions or anything. It's just a marker trait.

How about implementing `glib::UnsafeFrom<u32> for char` and requiring that in the `Unichar` trait?